apps/health/lib/health/checkers/real_time.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Health.Checkers.RealTime do
+  @moduledoc """
+  Health check which makes sure that real-time data (vehicle positions and
+  predictions) aren't stale.
+  """
+  @stale_data_seconds 15 * 60
+  @timestamps_fetcher &State.Metadata.updated_timestamps/0
```

Comment:
Come to think of it, do we need to inject this? We could call `State.Metadata.state_updated` from the tests, kind of like how we use other state modules' `new_state` functions.

Reply:
Hmm, personally I kind of like tests that don't depend too much on the state of other processes, but it does look like there's a bit of a convention of using the approach you describe with the various `State` modules. I'll give that a shot.